documentation/examples/prometheus-dockerswarm.yml

```diff
@@ -0,0 +1,36 @@
+# A example scrape configuration for running Prometheus with
+# Docker Swarm.
+#
+# This exemple works with Prometheus node_exporter deployed with:
+# docker service create --name node -l prometheus-job=node \
```

Comment:
Hmm, do we want to be encouraging people to run the node exporter in Docker? @SuperQ

Reply:
Similar to Kubernetes, for swarm users, this is likely to be common. Having correct documentation examples here is probably better than many of the incorrect docker-compose.yml configs I've seen from users having trouble with it.

Reply:
I did a bit more digging, and it looks like the big problem is that Swarm mode doesn't support `--pid=host`, so I'm pretty sure this won't work correctly. :frowning:

Reply:
updated for cadvisor